test/unit/file-list.spec.js

```diff
@@ -596,13 +596,13 @@ describe('FileList', () => {
       emitter.on('file_list_modified', modified)
 
       return list.refresh().then((files) => {
-        mockFs._touchFile('/some/b.js', '2020-01-01')
+        mockFs._touchFile('/some/b.js', '3020-01-01')
```

Comment:
this is advancing the future time by 1000 years. The commit message needs to be updated. Please also update the PR description.